blivet/populator/helpers/boot.py

```diff
@@ -59,6 +59,7 @@ def match(cls, data, device):
         fmt = formats.get_format(cls._type_specifier)
         try:
             return (super(MacEFIFormatPopulator, MacEFIFormatPopulator).match(data, device) and
+                    device.disk.format.supports_names and
```

Comment:
Is it possible this could be called with a non-partition device? If so I think it will crash. You could also use `device.disks[0]` which would work for any device type.

Reply:
`BootFormatPopulator.match` checks if the device is a partition so it should be ok.